src/index.md

```diff
@@ -9,20 +9,107 @@ of [Julia programming language](https://julialang.org/) ecosystem.
+* [Documenter.jl -- A documentation generator for Julia.](https://github.com/JuliaDocs/Documenter.jl)
+* [DocumenterCitations.jl -- DocumenterCitations.jl uses Bibliography.jl to add support for BibTeX citations and references in documentation pages generated by Documenter.jl.](https://github.com/JuliaDocs/DocumenterCitations.jl)
+* [DocumenterInterLinks.jl -- A plugin for Documenter.jl that enables linking between projects](https://github.com/JuliaDocs/DocumenterInterLinks.jl)
+* [DocumenterInventoryWritingBackport.jl -- Backport of inventory writing for Documenter 0.25-1.2](https://github.com/JuliaDocs/DocumenterInventoryWritingBackport.jl)
+* [DocumenterMarkdown.jl -- Documenter's Markdown / MkDocs backend](https://github.com/JuliaDocs/DocumenterMarkdown.jl)
+* [DocumenterMermaid.jl -- Mermaid.js diagram intergration for Documenter.jl](https://github.com/JuliaDocs/DocumenterMermaid.jl)
+* [DocumenterTools.jl -- Extra tools for setting up Documenter](https://github.com/JuliaDocs/DocumenterTools.jl)
+* [Highlights.jl -- A source code highlighter for Julia.](https://github.com/JuliaDocs/Highlights.jl)
+* [IOCapture.jl -- Capturing standard output and error streams in Julia.](https://github.com/JuliaDocs/IOCapture.jl)
+* [Julia-Cheat-Sheet -- Julia Cheat Sheet](https://github.com/JuliaDocs/Julia-Cheat-Sheet)
+* [LiveServer.jl -- Simple development server with live-reload capability for Julia.](https://github.com/JuliaDocs/LiveServer.jl)
+* [MarkdownAST.jl -- Abstract syntax tree representation of Markdown documents in Julia](https://github.com/JuliaDocs/MarkdownAST.jl)
+* [ModuleDocstrings.jl -- ](https://github.com/JuliaDocs/ModuleDocstrings.jl)
 
 ## Packages
 
 ```@eval
 import Main: PackageDefinition, package_table_markdown
 [
+PackageDefinition(
+    "ANSIColoredPrinters", "https://github.com/JuliaDocs/ANSIColoredPrinters.jl",
+    [
+        "README" => "https://github.com/JuliaDocs/ANSIColoredPrinters.jl",
+    ],
+    [
+        "https://codecov.io/gh/JuliaDocs/ANSIColoredPrinters.jl/branch/main/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/ANSIColoredPrinters.jl",
+    ]
+),
+PackageDefinition(
+    "Changelog", "https://github.com/JuliaDocs/Changelog.jl",
+    [
+        "README" => "https://github.com/JuliaDocs/Changelog.jl",
+    ],
+    [
+        "https://codecov.io/gh/JuliaDocs/Changelog.jl/branch/master/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/Changelog.jl",
+    ]
+),
+PackageDefinition(
+    "CodeEvaluation", "https://github.com/JuliaDocs/CodeEvaluation.jl",
+    [
+        "README" => "https://github.com/JuliaDocs/CodeEvaluation.jl",
+    ],
+    [
+        "https://codecov.io/gh/JuliaDocs/CodeEvaluation.jl/branch/main/graph/badge.svg" => "https://codecov.io/gh/JuliaDocs/CodeEvaluation.jl",
+    ]
+),
+PackageDefinition(
+    "DemoCards", "https://github.com/JuliaDocs/DemoCards.jl",
+    [
+        "README" => "https://github.com/JuliaDocs/DemoCards.jl",
```

Comment:
has docs, url just needs to be set as homepage URL in the github metadata

Reply:
Done.